src/tic_tac_toe/announcer.clj

```diff
@@ -0,0 +1,22 @@
+(ns tic-tac-toe.announcer)
+
+(def human-move "Please pick a spot from 1-%d: ")
+(def computer-move "The computer is picking a spot from 1-%d.")
```

Comment:
I don't think we need to announce that the computer is selecting a valid move

Reply:
OK, moved stuff around [here](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-25c91f67a97e706c5a657b280e537c55R9) and [here](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-954fe644a82dba16ad084dd0bdc1378cR11).  I wasn't sure if I'd like removing it, but it looks better, especially since I announce player turns.